scripts/linters/html_linter.py

```diff
@@ -110,6 +115,14 @@ def handle_starttag(self, tag, attrs):
             column_number + len(tag) + 2)
         starttag_text = self.get_starttag_text()
 
+        if self._SPACE_AROUND_ATTRIBUTE_REGEX.search(starttag_text):
```

Comment:
How about the following idea:

1. Splitting tag line(s) with space and none of the elements should start or end with `=`?
2. For each [attr, value] pair, the starttag_text should contain attr=value string?

Not sure whether this idea will work for all the cases but wanted to check whether we can use these instead of regex? [The currently used regex above is very hard to understand or maintain.]

Reply:
Should I try replacing the regex search and implement the same using splitting the starttag_text?  I mean try to implement the same logic programmatically.